graph_creator/static/graph_creator/graphs.js

```diff
@@ -0,0 +1,101 @@
+(function() {
+var d3 = Plotly.d3;
+
+var WIDTH_IN_PERCENT_OF_PARENT = 80,
+    HEIGHT_IN_PERCENT_OF_PARENT = 35;
+
+// Instances
+var instances_gd3 = d3.select('#instances')
+    .style({
+        width: WIDTH_IN_PERCENT_OF_PARENT + '%',
+        'margin-left': (100 - WIDTH_IN_PERCENT_OF_PARENT) / 2 + '%',
+
+        height: HEIGHT_IN_PERCENT_OF_PARENT + 'vh',
+        'margin-top': (35 - HEIGHT_IN_PERCENT_OF_PARENT) / 2 + 'vh'
```

Comment:
hm, `35-HEIGHT_IN_PERCENT_OF_PARENT` is `0`, because `HEIGHT_IN_PERCENT_OF_PARENT  === 35`. What is the reason of this formula?